modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -0,0 +1,147 @@
+:_mod-docs-content-type: PROCEDURE
+[id="builds-running-entitled-builds-with-sharedsecret-objects_{context}"]
+= Running builds using SharedSecret objects
+
+You can use a `SharedSecret` object to securely access the entitlement keys of a cluster in builds.
+
+The `SharedSecret` object allows you to share and synchronize secrets across namespaces.
+
+[IMPORTANT]
+====
+The Shared Resource CSI Driver feature is now generally available in link:https://docs.redhat.com/en/documentation/builds_for_red_hat_openshift/1.1[{builds-v2title} 1.1]. This feature is now deprecated in {product-title}. To use this feature, ensure you are using {builds-v2title} 1.1 or a more recent version.
+====
+
+.Prerequisites
+
+* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling features using feature gates].
+* You must have permission to perform the following actions:
+** Create build configs and start builds.
+** Discover which `SharedSecret` custom resource (CR) instances are available by entering the `oc get sharedsecrets` command and getting a non-empty list back.
+** Determine if the `builder` service account available to you in your namespace is allowed to use the given `SharedSecret` CR instance. In other words, you can run `oc adm policy who-can use <identifier_of_specific_SharedSecret>` to see if the `builder` service account in your namespace is listed.
```

Comment:
@sayan-biswas @adambkaplan is there any specific dependency on `builder` SA now (with builds operator)?

Reply:
No specific dependency on the `builder` service account. This can apply to any service account.

```suggestion
** Determine if any service account in your namespace is allowed to use the given `SharedSecret` instance. In other words, you can run `oc adm policy who-can use <identifier_of_specific_SharedSecret>` to see if the desired service account in your namespace is listed.
```